packages/sources/react/src/components/navigation/VtmnTabs/VtmnTabsItem.tsx

```diff
@@ -24,44 +18,28 @@ export interface VtmnTabsItemProps
    */
   badgeValue?: number;
 
-  /**
-   * Check whether the tabs item is currently selected or not.
-   * @defaultValue null
-   */
-  selected?: boolean;
-
-  onClick?: React.MouseEventHandler;
-
   /**
    * The content to render inside the component.
    * @defaultValue undefined
    */
   children?: React.ReactNode;
 }
 
-export const VtmnTabsItem = ({
-  icon,
-  href = '#',
-  badgeValue,
-  selected = false,
-  children,
-  onClick,
-  ...props
-}: VtmnTabsItemProps) => {
-  const handleClick = (e: React.MouseEvent<HTMLElement>) => {
-    e.preventDefault();
-    if (onClick) {
-      onClick();
-    }
-  };
-
+export const VtmnTabsItem = React.forwardRef<
+  HTMLButtonElement,
+  VtmnTabsItemProps
+>(({ icon, badgeValue, children, ...props }, ref) => {
   return (
     <li role="tab">
```

Comment:
There's a inconsistency here. In Svelte & Vue implementations, you put `role="tab"` on the `<button>`, not `<li>`.